rotkehlchen/utils/misc.py

```diff
@@ -370,3 +371,12 @@ def shift_num_right_by(num: int, digits: int) -> int:
 def is_valid_ethereum_tx_hash(val: str) -> bool:
     """Validates an Ethereum transaction hash."""
     return len(val) == 66 and is_hexstr(val) is True
+
+
+def create_order_by_rules_list(
+    data: Dict[str, Any],
+    timestamp_field: str = 'timestamp',
+) -> List[Tuple[str, bool]]:
```

Comment:
Please please please @yabirgb DO NOT make me repeat myself.

**Configure your editor to indent this properly**

Also add a docstring.